src/components/tag-input/tag-input.component.less

```diff
@@ -109,6 +109,12 @@ ux-tag-input {
 
     &.disabled {
         background-color: @tag-input-disabled-bg;
+
+        .ux-tag-icon,
+        .ux-select-icon {
+            cursor: default;
```

Comment:
On multiple select I still see the pointer cursor when disabled?

Reply:
That's it updated now